lib/components/SelectionControl/checkbox/checkbox-single/index.tsx

```diff
@@ -164,13 +165,18 @@ export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
                         {label}
                     </Text>
                 </label>
-                {showInfoIcon && (
-                    <InfoIcon
-                        className={clsx(
-                            "quill-checkbox__info-icon",
-                            infoIconClassName,
-                        )}
-                    />
+                {infoIconMessage && (
+                    <Tooltip
+                        shouldCloseToolTipOnMouseLeave
+                        tooltipContent={infoIconMessage}
+                    >
```

Comment:
Do we need to allow consumer to change the tooltip position or is it fixed in the design?

Reply:
it fixed as per figma in smart trader. if needed in future, tooltip position can be added.